variables.md

```diff
@@ -113,6 +118,62 @@ PERF_KERNEL | boolean | false | Enables kernel performance testing.
+PUBLIC_CLOUD | boolean | false | All Public Cloud tests have this variable set to true. Contact: qa-c@suse.de
+PUBLIC_CLOUD_ACCOUNT | string | "" | 
+PUBLIC_CLOUD_ACCNET | boolean | false | If set, az_accelerated_net test module is added to the job.
+PUBLIC_CLOUD_ARCH | string | "x86_64" | The architecture of created VM.
+PUBLIC_CLOUD_AZURE_OFFER | string | "" | 
+PUBLIC_CLOUD_AZURE_SKU | string | "" | 
+PUBLIC_CLOUD_BUILD | string | "" | The image build number. Used only when we use custom built image.
+PUBLIC_CLOUD_BUILD_KIWI | string | "" | The image kiwi build number. Used only when we use custom built image.
+PUBLIC_CLOUD_CHECK_BOOT_TIME | boolean | false | If set, boottime test module is added to the job.
+PUBLIC_CLOUD_CLIENT_ID | string | "" | 
+PUBLIC_CLOUD_CONFIDENTIAL_VM | boolean | false | GCE Confidential VM instance
+PUBLIC_CLOUD_CONSOLE_TESTS | boolean | false | If set, console tests are added to the job.
+PUBLIC_CLOUD_CONTAINERS | boolean | false | If set, containers tests are added to the job.
+PUBLIC_CLOUD_EC2_UPLOAD_AMI | string | "" | 
+PUBLIC_CLOUD_EC2_UPLOAD_SECGROUP | string | "" | 
+PUBLIC_CLOUD_EC2_UPLOAD_VPCSUBNET | string | "" | 
+PUBLIC_CLOUD_FIO | boolean | false | If set, storage_perf test module is added to the job.
+PUBLIC_CLOUD_FIO_RUNTIME | integer | 300 | Set the execution time for each FIO tests.
+PUBLIC_CLOUD_FIO_SSD_SIZE | string | "100G" | Set the additional disk size for the FIO tests.
+PUBLIC_CLOUD_GCE_UPLOAD_GUEST_FEATURES | string | "" | The list of features given to upload VM
+PUBLIC_CLOUD_IGNORE_EMPTY_REPO | boolean | false | Ignore empty maintenance update repos
+PUBLIC_CLOUD_IMAGE_ID | string | "" | The image ID we start the instance from
+PUBLIC_CLOUD_IMAGE_LOCATION | string | "" | The URL where the image gets downloaded from. The name of the image gets extracted from this URL.
+PUBLIC_CLOUD_IMAGE_PROJECT | string | "" | Google Compute Engine image project
+PUBLIC_CLOUD_IMG_PROOF_TESTS | string | false | Tests run  by img-proof. (We use 'default')
+PUBLIC_CLOUD_INSTANCE_TYPE | string | "" | Specify the instance type. Which instance types exists depends on the CSP. (default-azure: Standard_A2, default-ec2: t2.large )
+PUBLIC_CLOUD_KEY | string | "" | 
+PUBLIC_CLOUD_KEY_ID | string | "" | The CSP credentials key-id to used to access API.
+PUBLIC_CLOUD_KEY_SECRET | string | "" | The CSP credentials secret used to access API.
+PUBLIC_CLOUD_LTP | boolean | false | If set, the run_ltp test module is added to the job.
+PUBLIC_CLOUD_NO_CLEANUP_ON_FAILURE | boolean | false | Do not remove the instance when the test fails
+PUBLIC_CLOUD_PERF_DB_URI | string | "" | Optional variable. If set, the bootup times get stored in the influx database. The database name is 'publiccloud'. (e.g. PUBLIC_CLOUD_PERF_DB_URI=http://openqa-perf.qa.suse.de:8086)
+PUBLIC_CLOUD_PREPARE_TOOLS | boolean | false | Activate prepare_tools test module by setting this variable.
+PUBLIC_CLOUD_PROJECT_ID | string | "" | 
```

Comment:
This is only used by GCE lib again.

Reply:
also I would add some description or drop it ...

Reply:
```suggestion
PUBLIC_CLOUD_PROJECT_ID | string | "" | GCP only, used to specify the project id.
```

Reply:
in most places we use GCE , so let's stick to this or I need better understanding where I shall use GCE and where GCP .
CC: @ilausuch